Error when installing Auth Package

**EDIT: community-discovered workaround :point_down: [below](https://github.com/invertase/react-native-firebase/issues/3206#issuecomment-586652831) - pin @react-native-community/cli-platform-android to 3.0.3 for now**
--

## Issue

I have a new fresh project made today with react native. When I install "@react-native-firebase/auth" I get a error when building the app for android. The issue is only with the auth package, cause I builded the app only with "@react-native-firebase/app" (I have it installed) and it worked. 

```
* What went wrong:
A problem occurred evaluating project ':react-native-firebase_auth'.
> Project with path ':@react-native-firebase_app' could not be found in project ':react-native-firebase_auth'.
```

## Project Files
#### `package.json`:
```json
{
  "name": "freelouEstabelecimento",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "pod": "cd ./ios && pod install && cd ..",
    "release": "cd ./android %% ./gradlew assemblerelease && cd ..",
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "@react-native-community/masked-view": "^0.1.6",
    "@react-native-firebase/app": "^6.3.3",
    "@react-native-firebase/auth": "^6.3.3",
    "@react-navigation/bottom-tabs": "^5.0.3",
    "@react-navigation/native": "^5.0.3",
    "@react-navigation/stack": "^5.0.3",
    "react": "16.9.0",
    "react-native": "0.61.5",
    "react-native-gesture-handler": "^1.5.6",
    "react-native-navigation-bar-color": "^1.0.0",
    "react-native-reanimated": "^1.7.0",
    "react-native-safe-area-context": "^0.7.3",
    "react-native-screens": "^2.0.0-beta.2",
    "react-native-svg": "^11.0.1",
    "react-native-touchable-scale": "^2.1.0"
  },
  "devDependencies": {
    "@babel/core": "^7.6.2",
    "@babel/runtime": "^7.6.2",
    "@react-native-community/eslint-config": "^0.0.5",
    "babel-jest": "^24.9.0",
    "eslint": "^6.5.1",
    "jest": "^24.9.0",
    "metro-react-native-babel-preset": "^0.56.0",
    "react-test-renderer": "16.9.0"
  },
  "jest": {
    "preset": "react-native"
  }
}
```

### Android
#### `android/app/build.gradle`:
```groovy
apply plugin: "com.android.application"

import com.android.build.OutputFile

/**
 * The react.gradle file registers a task for each build variant (e.g. bundleDebugJsAndAssets
 * and bundleReleaseJsAndAssets).
 * These basically call `react-native bundle` with the correct arguments during the Android build
 * cycle. By default, bundleDebugJsAndAssets is skipped, as in debug/dev mode we prefer to load the
 * bundle directly from the development server. Below you can see all the possible configurations
 * and their defaults. If you decide to add a configuration block, make sure to add it before the
 * `apply from: "../../node_modules/react-native/react.gradle"` line.
 *
 * project.ext.react = [
 *   // the name of the generated asset file containing your JS bundle
 *   bundleAssetName: "index.android.bundle",
 *
 *   // the entry file for bundle generation
 *   entryFile: "index.android.js",
 *
 *   // https://facebook.github.io/react-native/docs/performance#enable-the-ram-format
 *   bundleCommand: "ram-bundle",
 *
 *   // whether to bundle JS and assets in debug mode
 *   bundleInDebug: false,
 *
 *   // whether to bundle JS and assets in release mode
 *   bundleInRelease: true,
 *
 *   // whether to bundle JS and assets in another build variant (if configured).
 *   // See http://tools.android.com/tech-docs/new-build-system/user-guide#TOC-Build-Variants
 *   // The configuration property can be in the following formats
 *   //         'bundleIn${productFlavor}${buildType}'
 *   //         'bundleIn${buildType}'
 *   // bundleInFreeDebug: true,
 *   // bundleInPaidRelease: true,
 *   // bundleInBeta: true,
 *
 *   // whether to disable dev mode in custom build variants (by default only disabled in release)
 *   // for example: to disable dev mode in the staging build type (if configured)
 *   devDisabledInStaging: true,
 *   // The configuration property can be in the following formats
 *   //         'devDisabledIn${productFlavor}${buildType}'
 *   //         'devDisabledIn${buildType}'
 *
 *   // the root of your project, i.e. where "package.json" lives
 *   root: "../../",
 *
 *   // where to put the JS bundle asset in debug mode
 *   jsBundleDirDebug: "$buildDir/intermediates/assets/debug",
 *
 *   // where to put the JS bundle asset in release mode
 *   jsBundleDirRelease: "$buildDir/intermediates/assets/release",
 *
 *   // where to put drawable resources / React Native assets, e.g. the ones you use via
 *   // require('./image.png')), in debug mode
 *   resourcesDirDebug: "$buildDir/intermediates/res/merged/debug",
 *
 *   // where to put drawable resources / React Native assets, e.g. the ones you use via
 *   // require('./image.png')), in release mode
 *   resourcesDirRelease: "$buildDir/intermediates/res/merged/release",
 *
 *   // by default the gradle tasks are skipped if none of the JS files or assets change; this means
 *   // that we don't look at files in android/ or ios/ to determine whether the tasks are up to
 *   // date; if you have any other folders that you want to ignore for performance reasons (gradle
 *   // indexes the entire tree), add them here. Alternatively, if you have JS files in android/
 *   // for example, you might want to remove it from here.
 *   inputExcludes: ["android/**", "ios/**"],
 *
 *   // override which node gets called and with what additional arguments
 *   nodeExecutableAndArgs: ["node"],
 *
 *   // supply additional arguments to the packager
 *   extraPackagerArgs: []
 * ]
 */

project.ext.react = [
    entryFile: "index.js",
    enableHermes: false,  // clean and rebuild if changing
]

apply from: "../../node_modules/react-native/react.gradle"

/**
 * Set this to true to create two separate APKs instead of one:
 *   - An APK that only works on ARM devices
 *   - An APK that only works on x86 devices
 * The advantage is the size of the APK is reduced by about 4MB.
 * Upload all the APKs to the Play Store and people will download
 * the correct one based on the CPU architecture of their device.
 */
def enableSeparateBuildPerCPUArchitecture = false

/**
 * Run Proguard to shrink the Java bytecode in release builds.
 */
def enableProguardInReleaseBuilds = false

/**
 * The preferred build flavor of JavaScriptCore.
 *
 * For example, to use the international variant, you can use:
 * `def jscFlavor = 'org.webkit:android-jsc-intl:+'`
 *
 * The international variant includes ICU i18n library and necessary data
 * allowing to use e.g. `Date.toLocaleString` and `String.localeCompare` that
 * give correct results when using with locales other than en-US.  Note that
 * this variant is about 6MiB larger per architecture than default.
 */
def jscFlavor = 'org.webkit:android-jsc:+'

/**
 * Whether to enable the Hermes VM.
 *
 * This should be set on project.ext.react and mirrored here.  If it is not set
 * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
 * and the benefits of using Hermes will therefore be sharply reduced.
 */
def enableHermes = project.ext.react.get("enableHermes", false);

android {
    compileSdkVersion rootProject.ext.compileSdkVersion

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    defaultConfig {
        applicationId "com.freelouestabelecimento"
        minSdkVersion rootProject.ext.minSdkVersion
        targetSdkVersion rootProject.ext.targetSdkVersion
        versionCode 1
        versionName "1.0"
    }
    splits {
        abi {
            reset()
            enable enableSeparateBuildPerCPUArchitecture
            universalApk false  // If true, also generate a universal APK
            include "armeabi-v7a", "x86", "arm64-v8a", "x86_64"
        }
    }
    signingConfigs {
        debug {
            storeFile file('debug.keystore')
            storePassword 'android'
            keyAlias 'androiddebugkey'
            keyPassword 'android'
        }
    }
    buildTypes {
        debug {
            signingConfig signingConfigs.debug
        }
        release {
            // Caution! In production, you need to generate your own keystore file.
            // see https://facebook.github.io/react-native/docs/signed-apk-android.
            signingConfig signingConfigs.debug
            minifyEnabled enableProguardInReleaseBuilds
            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
        }
    }
    // applicationVariants are e.g. debug, release
    applicationVariants.all { variant ->
        variant.outputs.each { output ->
            // For each separate APK per architecture, set a unique version code as described here:
            // https://developer.android.com/studio/build/configure-apk-splits.html
            def versionCodes = ["armeabi-v7a": 1, "x86": 2, "arm64-v8a": 3, "x86_64": 4]
            def abi = output.getFilter(OutputFile.ABI)
            if (abi != null) {  // null for the universal-debug, universal-release variants
                output.versionCodeOverride =
                        versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
            }

        }
    }
}

dependencies {
    implementation fileTree(dir: "libs", include: ["*.jar"])
    implementation "com.facebook.react:react-native:+"  // From node_modules

    if (enableHermes) {
        def hermesPath = "../../node_modules/hermes-engine/android/";
        debugImplementation files(hermesPath + "hermes-debug.aar")
        releaseImplementation files(hermesPath + "hermes-release.aar")
    } else {
        implementation jscFlavor
    }
}

// Run this once to be able to run the application with BUCK
// puts all compile dependencies into folder libs for BUCK to use
task copyDownloadableDepsToLibs(type: Copy) {
    from configurations.compile
    into 'libs'
}

apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
apply plugin: 'com.google.gms.google-services'

```

#### `android/settings.gradle`:
```groovy
rootProject.name = 'freelouEstabelecimento'
apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
include ':app'
```

#### `MainApplication.java`:
```java
package com.freelouestabelecimento;

import android.app.Application;
import android.content.Context;
import com.facebook.react.PackageList;
import com.facebook.react.ReactApplication;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.soloader.SoLoader;
import java.lang.reflect.InvocationTargetException;
import java.util.List;

public class MainApplication extends Application implements ReactApplication {

  private final ReactNativeHost mReactNativeHost =
      new ReactNativeHost(this) {
        @Override
        public boolean getUseDeveloperSupport() {
          return BuildConfig.DEBUG;
        }

        @Override
        protected List<ReactPackage> getPackages() {
          @SuppressWarnings("UnnecessaryLocalVariable")
          List<ReactPackage> packages = new PackageList(this).getPackages();
          // Packages that cannot be autolinked yet can be added manually here, for example:
          // packages.add(new MyReactNativePackage());
          return packages;
        }

        @Override
        protected String getJSMainModuleName() {
          return "index";
        }
      };

  @Override
  public ReactNativeHost getReactNativeHost() {
    return mReactNativeHost;
  }

  @Override
  public void onCreate() {
    super.onCreate();
    SoLoader.init(this, /* native exopackage */ false);
    initializeFlipper(this); // Remove this line if you don't want Flipper enabled
  }

  /**
   * Loads Flipper in React Native templates.
   *
   * @param context
   */
  private static void initializeFlipper(Context context) {
    if (BuildConfig.DEBUG) {
      try {
        /*
         We use reflection here to pick up the class that initializes Flipper,
        since Flipper library is not available in release mode
        */
        Class<?> aClass = Class.forName("com.facebook.flipper.ReactNativeFlipper");
        aClass.getMethod("initializeFlipper", Context.class).invoke(null, context);
      } catch (ClassNotFoundException e) {
        e.printStackTrace();
      } catch (NoSuchMethodException e) {
        e.printStackTrace();
      } catch (IllegalAccessException e) {
        e.printStackTrace();
      } catch (InvocationTargetException e) {
        e.printStackTrace();
      }
    }
  }
}
```

#### `AndroidManifest.xml`:
```xml
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
  package="com.freelouestabelecimento">

    <uses-permission android:name="android.permission.INTERNET" />

    <application
      android:name=".MainApplication"
      android:label="@string/app_name"
      android:icon="@mipmap/ic_launcher"
      android:roundIcon="@mipmap/ic_launcher_round"
      android:allowBackup="false"
      android:theme="@style/AppTheme">
      <activity
        android:name=".MainActivity"
        android:label="@string/app_name"
        android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
        android:windowSoftInputMode="adjustResize">
        <intent-filter>
            <action android:name="android.intent.action.MAIN" />
            <category android:name="android.intent.category.LAUNCHER" />
        </intent-filter>
      </activity>
      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
    </application>

</manifest>
```


## Environment
**`react-native info` output:**
```
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-3615QM CPU @ 2.30GHz
    Memory: 915.63 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 13.7.0 - /usr/local/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.6 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 25, 27, 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-29 | Google APIs Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5900203
    Xcode: 11.2/11B52 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5
```

<!-- change `[ ]` to `[x]` to select an option(s) -->

- **Platform that you're experiencing the issue on**:
  - [x] **Android** but have not tested behavior on iOS
- **`react-native-firebase` version you're using that has this issue:**
  - `6.3.3`
- **`Firebase` module(s) you're using that has the issue:**
  - ` "@react-native-firebase/app", "@react-native-firebase/auth"`
- **Are you using `TypeScript`?**
  - `N`

I'm also facing this issue same issue 

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/home/user/projects/projectname/node_modules/@react-native-firebase/storage/android/build.gradle' line: 60

* What went wrong:
A problem occurred evaluating project ':@react-native-firebase_storage'.
> Project with path ':@react-native-firebase_app' could not be found in project ':@react-native-firebase_storage'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':@react-native-firebase_storage'.
> compileSdkVersion is not specified.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 27s
user@aj-user:~/projects/projectname/android$ ./gradlew assembleRelease

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/home/user/projects/projectname/node_modules/@react-native-firebase/storage/android/build.gradle' line: 60

* What went wrong:
A problem occurred evaluating project ':@react-native-firebase_storage'.
> Project with path ':@react-native-firebase_app' could not be found in project ':@react-native-firebase_storage'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':@react-native-firebase_storage'.
> compileSdkVersion is not specified.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 5s

--- a/packages/admob/android/build.gradle
+++ b/packages/admob/android/build.gradle
@@ -79,7 +79,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-ads"
   implementation "com.google.android.ads.consent:consent-library:${ReactNative.ext.getVersion("ads", "consent")}"

--- a/packages/analytics/android/build.gradle
+++ b/packages/analytics/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-analytics"
 }

--- a/packages/auth/android/build.gradle
+++ b/packages/auth/android/build.gradle
@@ -21,7 +21,7 @@ plugins {
 project.ext {
   set('react-native', [
     versions: [
-      android           : [
+      android : [
         minSdk    : 16,
         targetSdk : 28,
         compileSdk: 28,
@@ -30,8 +30,8 @@ project.ext {
         buildTools: "28.0.3"
       ],
 
-      firebase          : [
-        bom : "21.1.0",
+      firebase: [
+        bom: "21.1.0",
       ],
     ],
   ])
@@ -57,7 +57,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-auth"
 }

--- a/packages/crashlytics/android/build.gradle
+++ b/packages/crashlytics/android/build.gradle
@@ -58,7 +58,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   // don't use firebase bom here as crashlytics-ndk is not included in the bom, trying to partially use bom here fails
   implementation "com.crashlytics.sdk.android:crashlytics:${ReactNative.ext.getVersion("firebase", "crashlytics")}"
   // ndk not in Firebase BoM

--- a/packages/database/android/build.gradle
+++ b/packages/database/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-database"
 }

--- a/packages/dynamic-links/android/build.gradle
+++ b/packages/dynamic-links/android/build.gradle
@@ -57,7 +57,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-dynamic-links"
 }

--- a/packages/firestore/android/build.gradle
+++ b/packages/firestore/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-firestore"
 }

--- a/packages/functions/android/build.gradle
+++ b/packages/functions/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-functions"
 }

--- a/packages/iid/android/build.gradle
+++ b/packages/iid/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-iid"
 }

--- a/packages/in-app-messaging/android/build.gradle
+++ b/packages/in-app-messaging/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-inappmessaging-display"
 }

--- a/packages/messaging/android/build.gradle
+++ b/packages/messaging/android/build.gradle
@@ -68,7 +68,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-messaging"
 }

--- a/packages/ml-natural-language/android/build.gradle
+++ b/packages/ml-natural-language/android/build.gradle
@@ -66,7 +66,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-ml-natural-language"
 }

--- a/packages/ml-vision/android/build.gradle
+++ b/packages/ml-vision/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-ml-vision"
 }

--- a/packages/perf/android/build.gradle
+++ b/packages/perf/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-perf"
 }

--- a/packages/remote-config/android/build.gradle
+++ b/packages/remote-config/android/build.gradle
@@ -62,7 +62,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-config"
 }

--- a/packages/storage/android/build.gradle
+++ b/packages/storage/android/build.gradle
@@ -57,7 +57,13 @@ repositories {
 }
 
 dependencies {
-  api project(':@react-native-firebase_app')
+  if (findProject(':@react-native-firebase_app')) {
+    api project(':@react-native-firebase_app')
+  } else if (findProject(':react-native-firebase_app')) {
+    api project(':react-native-firebase_app')
+  } else {
+    throw new GradleException('Could not find the react-native-firebase/app package, have you installed it?')
+  }
   implementation platform("com.google.firebase:firebase-bom:${ReactNative.ext.getVersion("firebase", "bom")}")
   implementation "com.google.firebase:firebase-storage"
 }

